migrations/20240414170023_create_userProfile_table.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.createTable("userProfile", (table) => {
+    table.increments("id");
+    table
+      .integer("user_id")
+      .unique()
+      .unsigned()
+      .references("users.id")
+      .onDelete("CASCADE");
+    table.string("firstname").notNullable();
+    table.string("lastname").notNullable();
+    table.integer("age").notNullable();
+    table.string("gender").notNullable();
+    table.text("about myself");
```

Comment:
It's not good to use spaces in the field names; it's better to use snake case, such as 'about_myself'.